docs/release-notes/3.0/release-3.0.0.md

```diff
@@ -0,0 +1,140 @@
+# [ML.NET](http://dot.net/ml) 3.0.0
+
+## **New Features**
+- **Add the ability to use Object Detection using TorchSharp** ([#6605](https://github.com/dotnet/machinelearning/pull/6605)) - We have added a new deep learning model back by TorchSharp that lets you fine tune your own Object Detection model!
+- **Add SamplingKeyColumnName to AutoMLExperiment API** ([#6649](https://github.com/dotnet/machinelearning/pull/6649)) - You can now set the SamplingKeyColumnName when you are using AutoML. Thanks @torronen!
+- **Add Object Detection to AutoML Sweeper** ([#6633](https://github.com/dotnet/machinelearning/pull/6633)) - Added Object Detection to the AutoML Sweeper so now they can be used together.
+- **Add String Vector support to DataFrame** ([#6628](https://github.com/dotnet/machinelearning/pull/6628)) - Adds support for String Vectors in DataFrame. This also allows for Better IDataView <-> DataFrame conversions.
+- **Add AutoZero tuner to BinaryClassification** ([#6615](https://github.com/dotnet/machinelearning/pull/6615)) - Can now user AutoZero tuner in AutoML Binary Classification experiments.
+- **Added in fairness assessment and mitigation** ([#6539](https://github.com/dotnet/machinelearning/pull/6539)) - Support for fairness assessment and mitigation tool
+- **Added in Support for some Intel OneDal Algorithms** ([#6521](https://github.com/dotnet/machinelearning/pull/6521)) - You can now use Intel's OneDal for some algorithms. This gives you access to some accelerated versions of these algorithms. The models are fully interoperable between ML.NET's normal models and these, so you can train with OneDal and then still run on machines where OneDal is not supported. Thanks @rgesteve!
+- **Add in ability to have pre-defined weights for ngrams** ([#6458](https://github.com/dotnet/machinelearning/pull/6458)) - If you know the weights of your NGrams already you can now directly provide that.
+- **Add SentenceSimilarity sweepable estimator in AutoML** ([#6445](https://github.com/dotnet/machinelearning/pull/6445)) - Can now use SentenceSimilarity with the sweepable estimator.
+- **Add VBufferDataFrameCoumn to DataFrame** ([#6409](https://github.com/dotnet/machinelearning/pull/6409)) - Now DataFrame can support the VBuffer from ML.NET so the IDataView <-> DataFrame conversion can work with those types.
+- **Added ADO.NET importing/exporting functionality to DataFrame** ([#5975](https://github.com/dotnet/machinelearning/pull/5975)) - Can now use ADO.NET import/export with DataFrames. Thanks @andrei-faber!
+- **Added in native binaries for Windows Arm64** ([#6813](https://github.com/dotnet/machinelearning/pull/6813))
+- **Switches some computational code to use the new Tensor Primitives package** ([#6875](https://github.com/dotnet/machinelearning/pull/6875))
+- **Add QA sweepable estimator in AutoML ** ([#6781](https://github.com/dotnet/machinelearning/pull/6781))
```

Comment:
This isn't rendering right in markdown. Remove the space after AutoML to get the bold working properly.

Reply:
```suggestion
- **Add QA sweepable estimator in AutoML** ([#6781](https://github.com/dotnet/machinelearning/pull/6781))
```

Reply:
Good catch.